### ISSUE TITLE
[31] Role and state are missing or applied inappropriately

### ISSUE THREAD

**Issue by bjgordon** [AusDTO/dta-website#304]

**Comment by bjgordon** [comment]
For @klepas to look at

**Comment by bjgordon** [comment]
Discussed this further with the tester. The use of <details> is not widely supported in screen-readers yet, so it is better to use a separate button with appropriatae roles as per recommendation. This is something for uikit.

**Comment by klepas** [comment]
Worth noting we provide support for browsers that don’t understand `<details>` via polyfills.

**Comment by bjgordon** [comment]
Given we've used the approach from UIKit which has been tested with users, and we have fallbacks for screen-readers that do not support `<details>`, I think for now we'll just have to defer to their advice on this issue.
